app/views/admin/users/show/emails/_form.html.erb

```diff
@@ -1,10 +1,12 @@
 <h2 class="crayons-subtitle-1">Send an email</h2>
+<p class="fs-m fw-normal mb-2">To: <%= @user.name %> (<%= @user.email %>)</p>
+<%= render "admin/users/show/emails/verification" %>
```

Comment:
I moved the rendering of the `verification` partial here and refactored some of the existing layout since we do not have updated copy that accounts for most of this code and since it more closely follows the design. If this is overkill, I will happily revert these changes until we figure out if we want to keep the email verification section as is and what we want the copy to be.

cc @Ridhwana

Reply:
I've asked Pawel for clarity [here](https://forem-team.slack.com/archives/C01S4QZUDNC/p1645772891174809).

Reply:
@juliannatetreault apologies, but let's revert this change and only make the copy changes as per https://forem-team.slack.com/archives/C01S4QZUDNC/p1645779370621179?thread_ts=1645772891.174809&cid=C01S4QZUDNC

Reply:
Will do, @Ridhwana!

Reply:
Done! ✔️